.gitignore

```diff
@@ -50,3 +50,4 @@ out/
 **/mainGeneratedDataTemplate
 **/mainGeneratedRest
 gobblin-dist*
+gobblin.tar.gz
```

Comment:
I think `gobblin-compaction` has a task to generate `gobblin.tar.gz`. OK, so ignore my comment on this.